### DISCUSSION TITLE
QT Files, so you don't need yo install the Apple program.

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#561] [General]
Here: 👉 https://gofile.io/d/zSCPLU 👈

📁 Put folder in: **E:\StaxRip-x64-2.1.8.4-Beta\Apps\Audio\qaac**

I don't know why this don't come included in new versions of StaxRip.

**Comment by Dendraspis** [comment]
> I don't know why this don't come included in new versions of StaxRip.

Simply because nobody wants to have problems because (s)he ignored the given licences. 😉 

> 📁 Put folder in: **E:\StaxRip-x64-2.1.8.4-Beta\Apps\Audio\qaac**

Does this path really work for everybody?  😲 

But I think some people are going to be satisfied with your files, especially when they are lazy and/or unable to extract them by themselves.

**Reply by ZeroCool22** [reply]
> Simply because nobody wants to have problems because (s)he ignored the given licences. 😉

Lets keep that as a secret between you an me ok? 😏

**Reply by stax76** [reply]
If the user clicks the download button in the apps dialog Apple Application Support page, the following wiki page is shown:

https://github.com/staxrip/staxrip/wiki/qaac
